src/provisioning.md

```diff
@@ -0,0 +1,24 @@
+# Endorsement and Reference Value Provisioning
```

Comment:
Since it is a book, I feel, we should have a similar markdown on Attestation Evidence and what are the current Evidence Formats supported by Veraison.

We can start with Base EAT and then describe the profiles, PSA Evidence and CCA Evidence as well, pointing to their profile documentation.

Reply:
And also the EAT library which Veraison maintains.

This is an important part of the RATS story

Reply:
Ignore I thin, you have covered Attestation Evidence, just realised this!